src/app/components/event/eventView.html

```diff
@@ -1,11 +1,13 @@
 <div class="container-fluid">
```

Comment:
Honestly, we should probably make all this 2-space tabs like the rest of the app, but that would be cruel to make you do that.